INSTALL

```diff
@@ -67,6 +67,20 @@ frame pointer based stack unwinder by passing the
 --enable-frame-pointers flag to configure.
 
 
+Building glog - Using vcpkg
```

Comment:
Please place the instructions after the basic installation section.

Reply:
Done.